src/components/Reports/GoalCalculator/CalculatorSettings/CalculatorSettings.tsx

```diff
@@ -0,0 +1,36 @@
+import SettingsIcon from '@mui/icons-material/Settings';
+import { useTranslation } from 'react-i18next';
+import {
+  GoalCalculatorCategory,
+  GoalCalculatorCategoryEnum,
+  GoalCalculatorStepEnum,
+} from '../GoalCalculatorHelper';
+import { InformationStep } from './Steps/InformationStep/InformationStep';
+import { OneTimeGoalsStep } from './Steps/OneTimeGoalsStep/OneTimeGoalsStep';
+import { SpecialIncomeStep } from './Steps/SpecialIncomeStep/SpecialIncomeStep';
+
+export const CalculatorSettings = (): GoalCalculatorCategory => {
```

Comment:
@wjames111 Thanks for adding localization! Since this function isn't a component, it shouldn't be capitalized. And since it calls a hook (`useTranslation`), it is now technically a hook itself. Can you rename it to `useCalculatorSettings`? You'll need to do the same thing for the other methods that return `GoalCalculatorCategory`. This can be done in another PR, just pointing it out.